mr/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -227,6 +227,7 @@ public void close() throws IOException {
 
     private CloseableIterable<T> open(FileScanTask currentTask, Schema readSchema) {
       DataFile file = currentTask.file();
+      LOG.debug("Opening [{}] for read", file);
```

Comment:
Aren't there already logs for this from the underlying file system implementation?

Reply:
You are right. Removed.